asf_core_data/pipeline/data_joining/merge_install_dates.py

```diff
@@ -157,34 +114,38 @@ def manage_hp_install_dates(df, identifier="UPRN", verbose=False):
 
     df["FIRST_HP_MENTION"] = df[identifier].map(dict(first_hp_mention))
 
-    df["ANY_HP"] = df[identifier].map(
-        dict(df.groupby(identifier)["HP_INSTALLED"].max())
-    )
+    print(df[df["FIRST_HP_MENTION"].notna() & df["HP_INSTALLED"]].shape)
```

Comment:
again, is this supposed to be here or just for a check?

Reply:
Yes, sorry, leftover from final testing.

Reply:
no worries, just flagging so that you know where these are at the end of testing :)